test/VariableToColumnMapTest.cpp

```diff
@@ -7,6 +7,7 @@
 #include "engine/VariableToColumnMap.h"
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
```

Comment:
```suggestion
#include <gmock/gmock.h>
#include <gtest/gtest.h>
```

I know, it's technically not part of this PR.